hubmapinventory/inventory.py

```diff
@@ -26,29 +27,31 @@
 
 
 ###############################################################################################################
-def __pprint(msg):
+def __pprint(msg: str):
```

Comment:
I haven't seen the double leading underscore pattern outside of a class--no change suggested, just curious what this denotes here.

Reply:
Just a little trick to hide functionality so that it doesn't appear in documentation when I use Sphinx.